Adyen/Analytics/Models/AnalyticsConstants.swift

```diff
@@ -12,6 +12,37 @@ public enum AnalyticsConstants {
     /// A constant to pass into the payment data object in the case where fetching the checkout attempt Id fails.
     public static let fetchCheckoutAttemptIdFailed = "fetch-checkoutAttemptId-failed"
     
+    /// Struct to hold error codes as type-safe static variables.
+    public struct ErrorCode {
+        
+        public static let redirectFailed = ErrorCode(600)
```

Comment:
There is a bunch of `code.stringValue` across this PR, is there a reason to make it Int instead of having an enum or just using a String raw value?

Reply:
It is a string on the backend unfortunately. I used static constants with a raw value Int, and a converted `stringValue`

Reply:
My point was more like why wouldn't we use string as rawValue? Having these as static let doesn't help with keeping them unique as it would be if using enum. This is how we can remove all data type conversions

Reply:
Do you mean using enums? With enums we would be creating an instance every single time. Since these are meant as constants, I made them static. Minor performance gain maybe but isn't it helpful?